README.md

```diff
@@ -1,361 +1,47 @@
-# Store 5
-
-## Why We Made Store
-
-- Modern software needs data representations to be fluid and always available.
-- Users expect their UI experience to never be compromised (blocked) by new data loads. Whether an
-  application is social, news or business-to-business, users expect a seamless experience both
-  online and offline.
-- International users expect minimal data downloads as many megabytes of downloaded data can quickly
-  result in astronomical phone bills.
-
-Store is a Kotlin library for loading data from remote and local sources.
-
-### Overview
-
-A Store is responsible for managing a particular data request. When you create an implementation of
-a Store, you provide it with a `Fetcher`, a function that defines how data will be fetched over
-network. You can also define how your Store will cache data in-memory and on-disk. Since Store
-returns your data as a `Flow`, threading is a breeze! Once a Store is built, it handles the logic
-around data flow, allowing your views to use the best data source and ensuring that the newest data
-is always available for later offline use.
-
-Store leverages multiple request throttling to prevent excessive calls to the network and disk
-cache. By utilizing Store, you eliminate the possibility of flooding your network with the same
-request while adding two layers of caching (memory and disk) as well as ability to add disk as a
-source of truth where you can modify the disk directly without going through Store (works best with
-databases that can provide observables sources
-like [Jetpack Room](https://developer.android.com/jetpack/androidx/releases/room)
-, [SQLDelight](https://github.com/cashapp/sqldelight)
-or [Realm](https://realm.io/products/realm-database/))
-
-### How to include in your project
-
-Artifacts are hosted on **Maven Central**.
-
-```kotlin
-STORE_VERSION = "5.0.0-alpha03"
-```
+<div align="center">
+    <img src="Images/friendly_robot.png" width="120"/>
+    <h1 style="font-size:48px">Store5</h1>
+</div>
+
+<div align="center">
+    <h4>Full documentation can be found on our <a href="https://mobilenativefoundation.github.io/Store/">website</a>!</h4>
+</div>
+
+## Concepts
+
+- [Store](https://mobilenativefoundation.github.io/Store/store/store/) is responsible for managing a particular read request
+- [MutableStore](https://mobilenativefoundation.github.io/Store/mutable-store/building/overview/) is
+  a [Store](https://mobilenativefoundation.github.io/Store/store/store/) that also manages particular write requests and
```

Comment:
Mutable repository implementation that allows (c)reate, (r)ead, (u)pdate and (d)elete  operations for local and network resources.